src/tools/miri/src/bin/miri.rs

```diff
@@ -67,7 +67,8 @@ impl rustc_driver::Callbacks for MiriCompilerCalls {
                 tcx.sess.fatal("miri cannot be run on programs that fail compilation");
             }
 
-            init_late_loggers(tcx);
+            let handler = EarlyErrorHandler::new(ErrorOutputType::default());
```

Comment:
Is it expected that a *second handler* gets created here? That seems odd and doesn't seem to happen with the other tools.

Reply:
This is not intentional and I think it was just a leftover from me trying to change stuff.